src/workloads/execution/CreateIndexSharded.yml

```diff
@@ -53,11 +132,6 @@ Actors:
       Key: CreateTextIndexesCmd
       Parameters:
         Duration: 5 minutes
-  - ExternalPhaseConfig:
```

Comment:
We cannot create unique indexes for an already-sharded collection. https://docs.mongodb.com/manual/core/index-unique/